niworkflows/interfaces/registration.py

```diff
@@ -347,8 +347,19 @@ def _post_run_hook(self, runtime):
 
 
 class EstimateReferenceImageInputSpec(BaseInterfaceInputSpec):
-    in_file = File(exists=True, mandatory=True, desc="4D EPI file")
-    sbref_file = File(exists=True, desc="Single band reference image")
+    in_file = traits.Either(File(exists=True),
+                            InputMultiPath(File(exists=True)),
```

Comment:
`InputMultiPath` already incorporates the possibility we may get one or more. Let's drop the `Either`.